server/src/test/java/org/elasticsearch/common/lucene/LuceneTests.java

```diff
@@ -563,14 +564,25 @@ public void testSortValueSerialization() throws IOException {
         Object sortValue = randomSortValue();
         Object deserialized = copyInstance(
             sortValue,
-            EMPTY_REGISTRY,
             Lucene::writeSortValue,
             Lucene::readSortValue,
             TransportVersionUtils.randomVersion(random())
         );
         assertEquals(sortValue, deserialized);
     }
 
+    private static <T> T copyInstance(T original, Writeable.Writer<T> writer, Writeable.Reader<T> reader, TransportVersion version)
```

Comment:
It's unclear to me what this method is for.  It's private, and I don't see where it's being called?  I'm probably just missing something...

Reply:
This method is to test the serialization / deserialization of SortValues. We cannot use the generic method because Sort values do not implement Writable. Instead we are using here:  Lucene::writeSortValue and Lucene::readSortValue.